### PR TITLE
[C-995] Remove message passing interfaces

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Removes message passing for all but the following:
- Oauth
- Account
- Reachability
- Some lifecycle items (mostly related to account)

These were left alone due to fixes already being in flight to avoid merge conflicts. We should follow up and remove everything once these land!

### CODE REVIEW

File: packages/mobile/src/components/delete-playlist-confirmation-drawer/DeletePlaylistConfirmationDrawer.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const DeletePlaylistConfirmationDrawer = () => {
 
   const handleDelete = useCallback(() => {
     if (playlistId) {
+      console.log('dispatching action??')
```
Comment (sliptype) [inline packages/mobile/src/components/delete-playlist-confirmation-drawer/DeletePlaylistConfirmationDrawer.tsx:21]: ??

File: packages/mobile/src/screens/app-screen/AppTabScreen.tsx [hunk 2]
```diff
@@ -120,26 +119,6 @@ export const AppTabScreen = ({ baseScreen, Stack }: AppTabScreenProps) => {
               swipeEnabled: isOnFirstTab
             })
           }
-        },
-        beforeRemove: (e) => {
```
Comment (sliptype) [inline packages/mobile/src/screens/app-screen/AppTabScreen.tsx:124]: Nice!

**Review by sliptype — APPROVED** [review]
image

File: packages/mobile/src/components/delete-playlist-confirmation-drawer/DeletePlaylistConfirmationDrawer.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/components/delete-playlist-confirmation-drawer/DeletePlaylistConfirmationDrawer.tsx:21]: ???????

File: packages/mobile/src/screens/app-screen/AppTabScreen.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/app-screen/AppTabScreen.tsx:124]: yeah that one felt great :D

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-995-remove-message-passing-2

**Pull request merged**